### PR TITLE
Support for Lua 5.1 and building on Mac OSX

### PR DESCRIPTION
**Pull request by tjo**

### CODE REVIEW

File: protobuf.rockspec [hunk 1]
```diff
@@ -9,7 +9,7 @@ description = {
   license = "MIT"
 }
 dependencies = {
-  "lua ~> 5.2"
+  "lua ~> 5.1"
```
Comment (djungelorm) [inline protobuf.rockspec]: Should be "lua ~> 5.1, ~> 5.2" as we support both 5.1 and 5.2?

File: Makefile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 .PHONY: all rockspec build clean
 
-VERSION=1.0.0
+VERSION=1.1.2
```
Comment (djungelorm) [inline Makefile]: Why the patch number increase? Can we set this to 1.1.0?

File: protobuf.rockspec (hunk 1 above)
Comment (tjo) [inline protobuf.rockspec]: You're right, but in this case we'll have to do "lua >= 5.1, < 5.3"

File: Makefile (hunk 2 above)
Comment (tjo) [inline Makefile]: Can do, I had to make a release for myself earlier, hence the major and patch. Will change.

**Pull request merged**